### PR TITLE
Add blog on Pulp Python release

### PR DESCRIPTION
**Pull request by melcorr**
@gerrod3 - I put this together with the primary objective of getting it into a Python newsletter where people mightn't know about Pulp. 

Can you let me know if anything seems a bit off to you? Or if I should mention anything that I havent?

### CODE REVIEW

File: _posts/2021-01-13-pulp-python-3.0-release.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Develop software safe in the knowledge that every time you add or remove content, Pulp creates an immutable repository version. If something doesn’t work out, you can roll back to the earlier version and continue, guaranteeing the safety and stability of your operation. Pulp maintains a record of all changes to help with troubleshooting and monitoring.
+
+## Distribute and promote content with ease
+
+Depending on your setup, content distribution can be laden with difficulties and require more than one tool. Pulp has been built with package distribution and promotion in mind. When the content that you’ve been developing is ready, you can easily distribute this content to a new environment for testing, staging and production.
+
+## Try Pulp Python 3.0 now!
+
+Pulp Python can be installed from [PyPi](https://pypi.org/project/pulp-python/3.0.0/) or [source](https://github.com/pulp/pulp_python/), click [here](https://pulp-python.readthedocs.io/en/latest/installation.html) for installation instructions. Client bindings for [Python](https://pypi.org/project/pulp-python-client/3.0.0/) and [Ruby]((https://rubygems.org/gems/pulp_python_client/versions/3.0.0)) are also available.
```
Comment (gerrod3) [inline _posts/2021-01-13-pulp-python-3.0-release.md]: I think the Ruby link has an extra pair of parenthesis.

Comment (gerrod3) [inline _posts/2021-01-13-pulp-python-3.0-release.md]: Also, I might have used 'PyPi' in my article but I think it is 'PyPI' like you used earlier.

**Review by gerrod3 — COMMENTED** [review]
I think everything else looks good with the post.

File: _posts/2021-01-13-pulp-python-3.0-release.md (hunk 1 above)
Comment (melcorr) [reply _posts/2021-01-13-pulp-python-3.0-release.md]: I copy pasted from you ;-) I'll update in your article also.

**Comment by melcorr** [comment]
Thanks @gerrod3

**Pull request merged**